frontend/app/.server/constants/service-identifier.contant.ts

```diff
@@ -0,0 +1,12 @@
+export const SERVICE_IDENTIFIER = {
+  CLIENT_CONFIG: Symbol.for('ClientConfig'),
+  CONFIG_FACTORY: Symbol.for('ConfigFactory'),
+  CONTAINER_CONFIG_PROVIDER: Symbol.for('ContainerConfigProvider'),
+  CONTAINER_PROVIDER: Symbol.for('ContainerProvider'),
+  CONTAINER_SERVICE_PROVIDER: Symbol.for('ContainerServicProvider'),
+  LOG_FACTORY: Symbol.for('LogFactory'),
+  PREFERRED_LANGUAGE_DTO_MAPPER: Symbol.for('PreferredLanguageDtoMapper'),
+  PREFERRED_LANGUAGE_REPOSITORY: Symbol.for('PreferredLanguageRepository'),
+  PREFERRED_LANGUAGE_SERVICE: Symbol.for('PreferredLanguageService'),
+  SERVER_CONFIG: Symbol.for('ServerConfig'),
+};
```

Comment:
Stealing @gregory-j-baker's suggestion from last week's meeting: I feel these constants should be declared within each interface rather than this constants file, which will be more difficult to maintain as we start introducing more components.

Reply:
Having them here helps to ensure there is no duplications. I have a tests that ensure the service identifier are unique.

Reply:
Fair point. Placing these service identifiers in their respective interfaces helps maintainability in my opinion. If for whatever reason an inject-able component is no longer needed, we may end up with unused constants if developers forget to remove them from this constants file.

Reply:
Make senses. I wonder if I can merge both solutions together.

Reply:
I will explore it in another PR.